Specs/Scene/LabelCollectionSpec.js

```diff
@@ -875,16 +948,23 @@ defineSuite([
                     continue;
                 }
 
-                width += billboard.width;
-                height = Math.max(height, billboard.height);
+                var glyphWidth = billboard.width * scale;
```

Comment:
This code is duplicate a lot; it is worth adding a utility function to this file?